13output.tex

```diff
@@ -206,22 +206,22 @@
 		\hline		
 		\endlastfoot
 		
-		SSB\_Unfished \Tstrut & Unfished reproductive potential (SSB is commonly female mature spawning biomass). \\
+		SSB\_Unfished \Tstrut & Unfished reproductive potential (gls{ssb} is commonly female mature spawning biomass). \\
```

Comment:
Should be SB not SSB. There is an inconsistency throughout the document of "spawning biomass" versus "spawning stock biomass" where I prefer the former but I guess I am just noting the inconsistency.

Reply:
I'm happy to switch to SB but I think in SS3 it's SSB throught. It might be confusing (or at least it could see it being confusing to people like me) to have SB in the manual and SSB in the output. Thoughts? @chantelwetzel-noaa

Reply:
I think keeping SSB in the manual to match the numerous occurrences in Report.sso makes sense to me. I'm happy to use SB in assessment reports and future assessment models, but it doesn't seem worth the effort to switch here.